src/root/layouts/application-root/application-root-layout.component.html

```diff
@@ -1,3 +1,83 @@
 <app-conteneur>
   <router-outlet></router-outlet>
+  <footer slot="footer" class="pt-5 pb-2" style="background-color: rgba(22, 22, 22, 1)">
+    <div class="container-fluid row">
+      <div class="flex-grow-1 col-auto">
+        <img src="/assets/img/footer-mark.svg" class="img-fluid" alt="" />
+      </div>
+      <div class="col-5">
+        <p class="text-secondary">
+          La Cartographie nationale des lieux de médiation numérique est un service fourni par l'Agence nationale de cohésion
+          des territoires
+          <span class="d-flex me-3">
+            <a class="nav-link me-1 text-muted" href="https://agence-cohesion-territoires.gouv.fr/" target="_blank">ANCT</a>
+            <i class="ri-external-link-line align-bottom text-muted"></i>
+          </span>
+        </p>
+        <p class="text-secondary">
+          Cette cartographie est dépendante du schéma de données des lieux de médiation numérique propulsé par la Mednum et
+          Datactivist
+          <span class="d-flex">
+            <a
+              class="nav-link me-1 text-muted"
+              href="https://lamednum.coop/schema-de-donnees-des-lieux-de-mediation-numerique-2/"
+              target="_blank">
```

Comment:
```suggestion
              target="_blank" rel="noopener noreferrer">
```
https://stackoverflow.com/questions/40593632/use-window-open-but-block-use-of-window-opener